css/style.css

```diff
@@ -0,0 +1,698 @@
+    position: relative;
+    box-shadow: 1px 0px 17px 0px rgba(0,0,0,0.75);
+    position: absolute;
+    left: -5rem;
+    top: 15rem;
+    transition: all 1s;
+    
+}
+.btn2{
+    top: 10rem;
+}
+.social-btn:hover{
+    left: 0;
+    background-color: #5D2D9E;
+}
+.social-btn a{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    text-decoration: none;
+    color: white;
+    font-weight: 600;
+    font-family:  'Hind', sans-serif;
+}
+.social-icon{
+    height: 3rem;
+    width: 3rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--secondary--color);
+    border-radius: 50%;
+    position: absolute;
+    right: 0px;
+}
+.github{
+    color:#5D2D9E;
+    font-size: 2rem;
+}
+
+.body-sec{
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column-reverse;
+    background-color: tomato;
+}
+.body-data{
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: rgb(255, 255, 255);
+    
+}
+.body-heading{
+    width: 50%;
+    padding-left: 10px;
+    font-size: 3.5rem;
+    line-height: 50px;
+    text-align: start;
+    font-weight: 600;
+    font-family: sans-serif;
+}
+
+.body-heading span{
+    font-size: 2.5rem;
+    color: #3499DC;
+}
+.body-img{
+    height: 50%;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    background-color: rgb(255, 255, 255);
+}
+.body-img img{
+    height: 60%;
+    width: 60%;
+    animation: side-img-animate infinite 4s linear;
+}
+.body-summary{
+    width: 60%;
+    padding-left: 6rem;
+    padding-top: 1rem;
+    color: rgb(144, 144, 144);
+    font-family: 'Hind',sans-serif;
+    font-size: 1.2rem;
+}
+.start{
+    height: 3rem;
+    font: 1.2em sans-serif;
+    width: 10rem;
+    cursor: pointer;
+    border-radius: 50px;
+    border: 2px solid #3499dc;
+    background-color: var(--secondary--color);
+    color: #3499dc;
+    margin-top: 1rem;
+    margin-left: 0 !important;
+    z-index: 200;
+    position: relative;
+
+}
+.launch-btn{
+    width: 48%;
+    position: relative;
+}
+.launch-btn img{
+    position: absolute;
+    z-index: -11;
+    height: 120%;
+    left: 0;
+    z-index: 0;
+    top: -2rem;
+    width: 40%;
+}
+.start:hover{
+    background-color: #3499dc;
+    color: var(--secondary--color);
+}
+
+.gear{
+    position: absolute;
+    top: 22%;
+    left: 0;
+    height: 3rem;
+    width: 3rem;
+    opacity: 0.5;
+    animation: App-logo-spin infinite 5s linear;
+}
+
+
+
+}
+
+
+
+@keyframes App-logo-spin {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+
+  @keyframes side-img-animate{         
+    0% {
+        transform: translate(0px,0px)
+    }
+    
+    
+    50% {
+        transform: translate(15px,5px)
+    }
+    
+    
+    100% {
+        transform: translate(0px,0px)
+    }	
+    }
```

Comment:
Lets break this into multiple files, for ease in review/read/modification. You can have 'baseStyle.css', 'projectName.css' to make more sense of split files.